src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/AnnotationUtils.java

```diff
@@ -85,4 +91,16 @@ public static boolean isAlleleSpecific(final InfoFieldAnnotation annotation) {
         }
         return Arrays.asList(rawDataString.split(ALLELE_SPECIFIC_SPLIT_REGEX));
     }
+
+    static String generateMissingDataWarning(final VariantContext vc, final Genotype g, final AlleleLikelihoods<GATKRead, Allele> likelihoods) {
+        final String positionString = "Annotation will not be calculated at position " + vc.getContig() + ":" + vc.getStart() +
+                " and possibly subsequent";
+        if (!g.isCalled()) {
+            return positionString + "; genotype for sample " + g.getSampleName() + " is not called";
+        } else if (likelihoods == null) {
+            return positionString +  "; alleleLikelihoodMap is null";
```

Comment:
Can it ever be the case that both `!g.isCalled()` and `likelihoods == null`? If so, you might want to change this `else if` to just `if` so that we can get both messages when both are true.